### PR TITLE
Add support for GitHub's CODEOWNERS files

### PR DESCRIPTION
**Pull request by jgraham**
https://github.com/blog/2392-introducing-code-owners

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/6506)

### CODE REVIEW

**Review by bobholt — APPROVED** [review]
wfm

**Review by sideshowbarker — APPROVED** [review]

**Review by sideshowbarker — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -338,7 +338,11 @@ everything that we can.
 OWNERS files are used only to indicate who should be notified of pull
 requests.  If you are interested in receiving notifications of proposed
 changes to tests in a given directory, feel free to add yourself to the
-OWNERS file. Anyone with expertise in the specification under test can
+OWNERS file. After changing an OWNERS file, you must run `wpt
```
Comment (gsnedders) [inline README.md]: This should be `./wpt`

**Comment by jgraham** [comment]
Note that this depends on #6421

**Comment by annevk** [comment]
Why not make  CODEOWNERS the source of truth?

**Comment by jgraham** [comment]
See public-test-infra https://lists.w3.org/Archives/Public/public-test-infra/2017JulSep/0000.html

**Comment by annevk** [comment]
I guess that's reasonable. Ping @tobie so he knows he can make simplifications to the bot.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/252000734)

# Firefox (nightly) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/252000735)

# Sauce (safari) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/252000741)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/252000742)

# Sauce (MicrosoftEdge) #

**Review by andreastt — APPROVED** [review]

**Comment by tobie** [comment]
Thanks for the ping, @annevk. I had completely forgotten about this PR (I was OoO when I was sent).

Wrote up some thoughts here: https://lists.w3.org/Archives/Public/public-test-infra/2017JulSep/0008.html. TL;DR: we can't both keep feature parity and simplify Ganesh given
the current GH CODEOWNERS implementation.

**Comment by jgraham** [comment]
Per @tobie's feedback this just adds complexity at the moment, so closing.

**Pull request closed without merging**